src/components/MeasurementExplorer.tsx

```diff
@@ -1,6 +1,7 @@
 /** @jsxImportSource @emotion/react */
 import { css } from '@emotion/react';
 import React, { useState } from 'react';
+import { CgArrowsExchangeAlt } from 'react-icons/cg';
```

Comment:
Can you please use either a unicode character, or import something from `react-icons/fa`. We don't want to use something than FontAwesome as icons.